src/config.ts

```diff
@@ -18,15 +20,43 @@ export async function loadConfig(): Promise<SyncConfig> {
   log('Resolving sync config file.');
 
   const filename = path.resolve(process.cwd(), CONFIG_FILENAME);
-  const migrations = await yaml.load(filename) as SyncConfig;
+  const loadedConfig = (await yaml.load(filename)) as SyncConfig;
 
   log('Resolved sync config file.');
 
-  const loaded = mergeDeepRight(DEFAULT_CONFIG, migrations) as SyncConfig;
+  const loaded = mergeDeepRight(DEFAULT_CONFIG, loadedConfig) as SyncConfig;
 
-  // TODO: Validate the loaded config.
+  validate(loaded);
 
-  return loaded;
+  return {
+    ...loaded,
+    injectedConfig: {
+      ...loaded.injectedConfig,
+      vars: prepareInjectionConfigVars(loaded.injectedConfig.vars)
+    }
+  };
+}
+
+/**
+ * Validate the loaded configuration.
+ *
+ * @param {SyncConfig} config
+ */
+export function validate(config: SyncConfig) {
+  const { injectedConfig } = config;
+
+  // Shouldn't reach under here unless the user has mismatched the value.
+  if (!injectedConfig.table || !injectedConfig.table.trim()) {
```

Comment:
Let's not allow the user to specify the table name? Because this is a temporary table that the user doesn't need to have access to.

Reply:
Makes sense.